Milvus.Client/FieldData.cs

```diff
@@ -120,22 +120,40 @@ internal static FieldData FromGrpcFieldData(Grpc.FieldData fieldData)
                 }
 
             case Grpc.FieldData.FieldOneofCase.Scalars:
-                return fieldData.Scalars.DataCase switch
+                return fieldData.Scalars switch
                 {
-                    Grpc.ScalarField.DataOneofCase.BoolData
+                    {DataCase: ScalarField.DataOneofCase.BoolData}
```

Comment:
nit (below as well):

```suggestion
                    { DataCase: ScalarField.DataOneofCase.BoolData }
```

Reply:
Code style changed